docs/linux/external_fuzzing_usb.md

```diff
@@ -216,9 +201,11 @@ cp bin/linux_arm/syz-executor ~/syz-bin/
     ``` bash
     $ cat usb.log
     r0 = syz_usb_connect(0x0, 0x24, &(0x7f00000001c0)={{0x12, 0x1, 0x0, 0x8e, 0x32, 0xf7, 0x20, 0xaf0, 0xd257, 0x4e87, 0x0, 0x0, 0x0, 0x1, [{{0x9, 0x2, 0x12, 0x1, 0x0, 0x0, 0x0, 0x0, [{{0x9, 0x4, 0xf, 0x0, 0x0, 0xff, 0xa5, 0x2c}}]}}]}}, 0x0)
-    $ sudo ./syz-bin/syz-execprog -executor ./syz-bin/syz-executor -threaded=0 -collide=0 -procs=1 -enable='' -debug usb.log
+    $ sudo ./syz-bin/syz-execprog -slowdown 3 -executor ./syz-bin/syz-executor -threaded=0 -collide=0 -procs=1 -enable='' -debug usb.log
```

Comment:
We should not recommend to use non default values for slowdown. It's pointless because fuzzing uses the default value. So if default value is wrong, this won't fix it. And it makes programs run not the way they run during fuzzing, which can make reproducers fail to reproduce bugs.

Reply:
The default timeouts work for fuzzing in a VM. This timeout is only for running reproducers. AFAIR remember, I made it larger, as the Raspberry Pi hardware for emulating USB devices is slower.

Anyway, this change improves the instructions we have right now. Proper support will need to be implemented separately.